Source/WebInspectorUI/UserInterface/Controllers/JavaScriptLogViewController.js

```diff
@@ -264,6 +264,8 @@ WI.JavaScriptLogViewController = class JavaScriptLogViewController extends WI.Ob
         };
 
         WI.runtimeManager.evaluateInInspectedWindow(text, options, printResult.bind(this));
+
+        WI.javaScriptRuntimeCompletionProvider.clearCachedPropertyNames();
```

Comment:
NIT: from an order of operations perspective, i think it'd be safer to clear the cached property names before `WI.runtimeManager.evaluateInInspectedWindow` just in case it does something that'd affect the cached property names (which i realize is not the case today, but could change in the future and result in a potentially very subtle bug)